CSS/style.css

```diff
@@ -0,0 +1,30 @@
+*
+{
```

Comment:
*{ }, this is the syntax i remember, using this one I have never seen this before.